keras_nlp/src/models/efficientnet_v1/efficientnet_v1_backbone_test.py

```diff
@@ -0,0 +1,252 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import keras
+import numpy as np
+import pytest
+from absl.testing import parameterized
+
+from keras_nlp.src.models.efficientnet_v1.efficientnet_v1_backbone import (  # noqa: E501
+    EfficientNetV1Backbone,
+)
+from keras_nlp.src.tests.test_case import TestCase
+from keras_nlp.src.utils.keras_utils import get_feature_extractor
+
+
+class EfficientNetV1BackboneTest(TestCase):
```

Comment:
Follow the form of other backbone tests that have been added. We have some common test routines.